apps/frontend/src/pages/auth/components/sign-in-form/sign-in-form.tsx

```diff
@@ -1,16 +1,89 @@
-import { Button } from "~/libs/components/components.js";
+import {
+	Button,
+	IconButton,
+	Input,
+	Link,
+} from "~/libs/components/components.js";
+import { AppRoute } from "~/libs/enums/enums.js";
+import { useAppForm, useCallback, useState } from "~/libs/hooks/hooks.js";
+import {
+	type UserSignInRequestDto,
+	userSignInValidationSchema,
+} from "~/modules/users/users.js";
+
+import { DEFAULT_SIGN_IN_PAYLOAD } from "./libs/constants/constants.js";
+import styles from "./styles.module.css";
 
 type Properties = {
-	onSubmit: () => void;
+	onSubmit: (payload: UserSignInRequestDto) => void;
 };
 
-const SignInForm = ({ onSubmit }: Properties): JSX.Element => (
-	<>
-		<h1>Sign In</h1>
-		<form onSubmit={onSubmit}>
-			<Button label="Sign in" />
-		</form>
-	</>
-);
+const SignInForm = ({ onSubmit }: Properties): JSX.Element => {
+	const { control, errors, handleSubmit } = useAppForm<UserSignInRequestDto>({
+		defaultValues: DEFAULT_SIGN_IN_PAYLOAD,
+		validationSchema: userSignInValidationSchema,
+	});
+
+	const [isPasswordVisible, setIsPasswordVisible] = useState<boolean>(false);
+
+	const handleFormSubmit = useCallback(
+		(event_: React.BaseSyntheticEvent): void => {
+			void handleSubmit((formData: UserSignInRequestDto) => {
+				onSubmit(formData);
+			})(event_);
+		},
+		[handleSubmit, onSubmit],
+	);
+
+	const handleTogglePasswordVisibility = useCallback(() => {
+		setIsPasswordVisible((previousState) => !previousState);
+	}, []);
+
+	return (
+		<section className={styles["auth-container"]}>
+			<div className={styles["left-side"]}>
+				{/* TODO: replace logo */}
+				<img alt="logo" className={styles["logo-wrapper"]} src="" />
+			</div>
+			<div className={styles["right-side"]}>
+				<h3 className={styles["form-title"]}>Welcome back</h3>
```

Comment:
```suggestion
```
These styles should be on the auth page. You should reuse them for sign-in/sign-up forms